ui/partials/perf/comparesubtestctrl.html

```diff
@@ -7,8 +7,7 @@
   <div id="subtest-summary" ng-if="!dataLoading && !errors.length">
     <h1>{{subtestTitle}} subtest summary</h1>
     <ul class="list-inline push-information">
-      <li><strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}})</li>
-      <li><strong>New</strong> - <a href="{{newProject.getRevisionHref(newRevision)}}">{{newRevision}}</a> ({{newProject.name}})</li>
+      <revision-information original-project="originalProject" original-revision="originalRevision" original-result-set="originalResultSet" new-project="newProject" new-revision="newRevision" new-result-set="newResultSet"></revision-information>
```

Comment:
This doesn't seem to work in my testing. 

e.g. http://localhost:8000/perf.html#/comparesubtest?originalProject=mozilla-central&originalRevision=9bca608ab65a&newProject=mozilla-central&newRevision=d3228c82badd&originalSignature=2b7ae2df2a55d4576b0ebc4b22747b3cfa419e8e&newSignature=2b7ae2df2a55d4576b0ebc4b22747b3cfa419e8e

Could you double check that everything is defined here?